projects/angular/src/data/datagrid/utils/key-navigation-grid.controller.ts

```diff
@@ -111,8 +112,9 @@ export class KeyNavigationGridController implements OnDestroy {
             const activeItem = this.rows
               ? (Array.from(this.rows[y].querySelectorAll(this.config.keyGridCells))[x] as HTMLElement)
               : null;
+            const scrollItem = true;
```

Comment:
There is a better way to do this. You can change the signature of the `setActiveCell` to take an object as the second parameter so that the boolean literal is named there instead of in a variable that is used once.